src/database/queries/update_events_query.js

```diff
@@ -0,0 +1,30 @@
+const Events = require('../event_schema');
+
+const query = (req, callback) => {
```

Comment:
And what does this `query` do exactly? Can't I understand it's purpose from the name? And what is the `req` it's taking as input?